client/my-sites/importer/section-import.jsx

```diff
@@ -271,7 +271,7 @@ class SectionImport extends Component {
 		const { jetpack: isJetpack } = site;
 		const headerText = translate( 'Import your content' );
 		const subHeaderText = translate(
-			'Bring content hosted elsewhere over to WordPress.com. ' +
+			'Bring content hosted elsewhere over to WordPress.com.' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Bring content hosted elsewhere over to WordPress.com. {{a}}Find out what we currently support.{{/a}}' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).